.eslintrc

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Using `.eslintrc` without the extension has been deprecated, we should probably update this to include `.eslintrc.json` https://eslint.org/docs/user-guide/configuring#configuration-file-formats